### DISCUSSION TITLE
Exiting the lightbox submits the form

### DISCUSSION THREAD

**Discussion by 7prog** [igordanchenko/yet-another-react-lightbox#235] [Q&A] [answered]
Closing the full screen causes the form to submit,  can we have a fix for that?

**Comment by igordanchenko** [comment]
Please provide a repro.

**Reply by 7prog** [reply]
In the  simple code below, when lightbox is fullscreen; if you close eather by the button or by echap key, the form will be submitted

```jsx
<form
    onSubmit={(e) => {
      console.log("submited");
    }}
>
  <Lightbox
    open={open_lightbox}
    close={() => {
      setOpen_lightbox(false);
    }}
    slides={images}
    className={cn("", classNames?.lightbox)}
    plugins={[Zoom]}
  />
  <button type="submit">submit</button>
</form>
```

**Reply by igordanchenko** [reply]
I'm afraid the above snippet does not reproduce the issue.

https://codesandbox.io/p/devbox/yet-another-react-lightbox-235-vhg8v7

Please provide a reproducible example.

**Reply by 7prog** [reply, marked as answer]
You are right! I found that a custom invisible container with wrong event handler trigger the submit. thank you

**Reply by igordanchenko** [reply]
I'm glad to hear you were able to identify the culprit.
